projects/core/src/lib/operators/dialogs/rasterization/rasterization.component.html

```diff
@@ -0,0 +1,79 @@
+        <geoengine-dialog-section-heading title="Config" subtitle="Specify the rasterization parameters">
+        </geoengine-dialog-section-heading>
+        <div formGroupName="rasterization">
+            <mat-tab-group
+                color="primary"
+                disablePagination
+                [selectedIndex]="selected.value"
+                (selectedIndexChange)="selected.setValue($event)"
+            >
+                <mat-tab label="Grid">
+                    <ng-container *ngTemplateOutlet="gridTemplate"></ng-container>
+                </mat-tab>
+                <mat-tab label="Density">
+                    <ng-container *ngTemplateOutlet="densityTemplate"></ng-container>
+                </mat-tab>
+            </mat-tab-group>
+
+            <ng-template #gridTemplate>
+                <label id="radio-group-label" class="labelRadioGroup">Size mode:</label>
+                <mat-radio-group color="primary" formControlName="gridSizeMode" aria-labelledby="radio-group-label">
+                    <mat-radio-button value="fixed">Fixed</mat-radio-button>
+                    <mat-radio-button value="relative">Relative</mat-radio-button>
+                </mat-radio-group>
+                <div formGroupName="resolution" class="groupInRow">
+                    <mat-form-field appearance="fill">
+                        <mat-label>Resolution (x)</mat-label>
+                        <input matInput type="number" formControlName="resX" />
+                    </mat-form-field>
+                    <mat-form-field appearance="fill">
+                        <mat-label>Resolution (y)</mat-label>
+                        <input matInput type="number" formControlName="resY" />
+                    </mat-form-field>
+                </div>
+                <div formGroupName="origin" class="groupInRow">
+                    <mat-form-field appearance="fill">
+                        <mat-label>Origin (x)</mat-label>
+                        <input matInput type="number" formControlName="originX" />
+                    </mat-form-field>
+                    <mat-form-field appearance="fill">
+                        <mat-label>Origin (y)</mat-label>
+                        <input matInput type="number" formControlName="originY" />
+                    </mat-form-field>
+                </div>
+            </ng-template>
+            <ng-template #densityTemplate>
+                <div class="groupInRow" *ngIf="selected.value === 1">
+                    <mat-form-field appearance="fill">
+                        <mat-label>Radius</mat-label>
+                        <input matInput type="number" formControlName="radius" />
+                    </mat-form-field>
+                    <mat-form-field appearance="fill">
+                        <mat-label>Stddev</mat-label>
+                        <input matInput type="number" formControlName="stddev" />
+                    </mat-form-field>
```

Comment:
These values should be positive.

Moreover, if there is the possibility to set the operator wrongly, clicking create does not result in any notification to the user that this failed.

Reply:
6f87f00